utils/crypto/envVars.ts

```diff
@@ -5,9 +5,9 @@ export const envVars = {
     NODEJS_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
 
     // dataproxy > cloudflare-workers
-    CF_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
-    CF_ACCOUNT_ID: "U2FsdGVkX1/kgKmF6Nc3rIIkben8az6boItdyzPJFj5vEPoxnWtfpyjEO8E6voHLB4Zl/+kddgTA14uboA+40A==",
-    CF_API_TOKEN: "U2FsdGVkX18fx8PpClVvhLV9TuBGm//20tdPUYRowj/hBUw+r7nECbfZ7DBPKgI9lelSXXKOXD4FFNg6FPpURA==",
+    CLOUDFLARE_DATA_PROXY_URL: "U2FsdGVkX187qRBSpPYoBu+9kcycYm/7BvLMBftPQbzKid8w0ZXamlDLESDjjlWZZoq40pyYNZxyOa7Y9jXul6U7lf2NPP5EZ41LS1kAW1nUGfSkFf/5Lgek5kbxoaOChuhboHmydnb/Ok5ztaVtLcTzYP4HmSeOAPEwDid9M7sD1CXcsmfA+1bDDpKmggFB",
+    CLOUDFLARE_ACCOUNT_ID: "U2FsdGVkX1/kgKmF6Nc3rIIkben8az6boItdyzPJFj5vEPoxnWtfpyjEO8E6voHLB4Zl/+kddgTA14uboA+40A==",
+    CLOUDFLARE_API_TOKEN: "U2FsdGVkX18fx8PpClVvhLV9TuBGm//20tdPUYRowj/hBUw+r7nECbfZ7DBPKgI9lelSXXKOXD4FFNg6FPpURA==",
```

Comment:
Curiosity question: was the env var change necessary just to turn the `CF` string into the more explicit `CLOUDFLARE`?

Reply:
It triggers a deprecated warning from wrangler CLI